shared-settings.gradle

```diff
@@ -0,0 +1,57 @@
+import org.yaml.snakeyaml.Yaml
```

Comment:
Let's add the license here.

Reply:
OK, Just saw our build scripts actually don't have one either, but I added it as the other settings.gradle files have one.